frontend/packages/data-portal/app/i18n.ts

```diff
@@ -58,13 +63,14 @@ export const i18n = {
   plusMore: (count: number) => `+${count} More`,
   poor: 'Poor',
   portalId: (id: number | string) => `Portal ID: ${id}`,
-  portalIdBlank: 'Portal ID:',
+  portalIdBlank: 'PORTAL ID',
```

Comment:
I'd like this to remain title case but use `text-transform: uppercase` in the css